orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -113,32 +111,27 @@ def _extract_input_information(module, *inputs, **kwargs):
 class ORTModule(torch.nn.Module):
 
     def __init__(self, module):
-        assert isinstance(module, torch.nn.Module), "'module' mst be a torch.nn.Module"
+        assert isinstance(module, torch.nn.Module), "'module' must be a torch.nn.Module"
         super(ORTModule, self).__init__()
 
-        self._export_again = False
         # TODO: This is incorrect when different layers may be in different devices
         self._device = next(module.parameters()).device
         self._device_changed = False
 
         # User module is wrapped to use its initializers and save computed gradients
         self._original_module = module
         self._onnx_training = None
+        self._is_training = True
 
         # Related to training graph split/shape inference
         self._current_input_shape = None
         self._module_gradient_graph_builder = None
-        self._input_names_require_grad = None
```

Comment:
Is this removal intentional? That was recently added to support the case where input which requires grad is provided by the user. In particular, we are using this on a DeepSpeed sample

Reply:
good catch. reverted.